### PR TITLE
Add configuration

### PR DESCRIPTION
**Pull request by PakhiAgarwal**

### CODE REVIEW

File: loader.py [hunk 1]
```diff
@@ -221,50 +226,58 @@ def save_summary(rate, replicateId, burnIn):
...
+    if not os.path.exists("out"):
         os.makedirs("out")
     else:
-        for filename in glob.glob("out/*summary*.csv"): os.remove(filename)
+        for filename in glob.glob("out/*summary*.csv"):
+            os.remove(filename)
 
     # Parse the parameters
+    configuration = sys.argv[1]
     studyId = int(sys.argv[1])
```
Comment (rjzupkoii) [inline loader.py:277]: Make sure you are testing the scripts, both of these variables are referring to the same index number.

File: loader.py [hunk 2]
```diff
@@ -221,50 +226,58 @@ def save_summary(rate, replicateId, burnIn):
...
+
         # Update the status for the user
         if len(studies) > 1:
             print("{} of {} studies complete".format(counter, len(studies)))
             counter += 1
 
 if __name__ == '__main__':
+
+
     if len(sys.argv) != 3:
```
Comment (rjzupkoii) [inline loader.py:261]: This value needs to be updated to match the number of parameters.

**Review by rjzupkoii — CHANGES_REQUESTED** [review]
Looks like the script needs to be updated to Python3 which should be straightforward. I suspect that some testing of the script is still needed, so take a closer look at that. Also, be sure to clear all of the items that the PEP8 script is mentioning.

**Comment by PakhiAgarwal** [comment]
@rjzupkoii I have resolved PEP8 conflicts and updated the script to Python3. Please let me know if it looks okay now.

File: loader.py [hunk 3]
```diff
@@ -45,13 +46,14 @@ def get_frequency_subset(replicateId, subset):
...
+            SELECT md.id, msd.locationid, msd.infectedindividuals
             FROM sim.monthlydata md
                 INNER JOIN sim.monthlysitedata msd ON msd.monthlydataid = md.id
-            WHERE md.replicateid = %(replicateId)s AND md.dayselapsed in ({})) two ON one.id = two.id AND one.locationid = two.locationid
+            WHERE md.replicateid = %(replicateId)s AND md.dayselapsed in ({})) two ON one.id = two.id
+            AND one.locationid = two.locationid
         INNER JOIN sim.location l on l.id = one.locationid
         GROUP BY dayselapsed, l.x, l.y, infectedindividuals""".format(subset, subset)
-    return select(sql, {'replicateId':replicateId, 'startDay':startDay})
+    return select(sql, {'replicateId': replicateId, 'startDay': startDay})
```
Comment (rjzupkoii) [inline loader.py:56]: Looks like the signature for the `select` functions still haven't been updated?

**Review by rjzupkoii — CHANGES_REQUESTED** [review]

**Comment by rjzupkoii** [comment]
Try using this command line when testing the loader script, you may need to pull the Rwanda repository (https://github.com/rjzupkoii/PSU-CIDD-Rwanda):

`loader.py rwa-configuration.yml 2 4015`

**Comment by PakhiAgarwal** [comment]
> Try using this command line when testing the loader script, you may need to pull the Rwanda repository (https://github.com/rjzupkoii/PSU-CIDD-Rwanda):
> 
> `loader.py rwa-configuration.yml 2 4015`

I cannot find `rwa-configuration.yml`.
Also in the current MaSim repo there is no yml file. Have you removed them from here?

**Comment by PakhiAgarwal** [comment]
Surprisingly, I am not getting any errors while running from Pycharm, but from terminal, there are a few errors.

**Comment by PakhiAgarwal** [comment]
> > Try using this command line when testing the loader script, you may need to pull the Rwanda repository (https://github.com/rjzupkoii/PSU-CIDD-Rwanda):
> > `loader.py rwa-configuration.yml 2 4015`
> 
> I cannot find `rwa-configuration.yml`.
> Also in the current MaSim repo there is no yml file. Have you removed them from here?

I found the file.

**Comment by PakhiAgarwal** [comment]
> Try using this command line when testing the loader script, you may need to pull the Rwanda repository (https://github.com/rjzupkoii/PSU-CIDD-Rwanda):
> 
> `loader.py rwa-configuration.yml 2 4015`

@rjzupkoii While running with these values, I am getting this error:
`TypeError: Expected bytes or unicode string, got dict instead`
Is it related to `psycopg`?

**Comment by rjzupkoii** [comment]
@PakhiAgarwal It's most likely related to the file access, try `with open(filename, "w") as csvfile:` instead.

**Comment by PakhiAgarwal** [comment]
> with open(filename, "w") as csvfile:

I tried this but its the same, I think the error is in this line `return select(connection, sql, {'studyId': studyId})`
The error is `TypeError: Expected bytes or unicode string, got dict instead`, I think it refers to `{'studyId': studyId})`
I am not sure what changes are required here.

**Comment by rjzupkoii** [comment]
@PakhiAgarwal Is there a stack trace being returned by Python that you can post?

**Comment by PakhiAgarwal** [comment]
> @PakhiAgarwal Is there a stack trace being returned by Python that you can post?

error_8

**Comment by rjzupkoii** [comment]
@PakhiAgarwal You're passing in the entire configuration when you should just be passing the connection string to the function.

**Comment by PakhiAgarwal** [comment]
> @PakhiAgarwal You're passing in the entire configuration when you should just be passing the connection string to the function.

Thanks. Got it. Now I am getting connection time out error. 
` Is the server running on host "masimdb.vmhost.psu.edu" (128.118.15.151) and accepting
        TCP/IP connections on port 5432?`

**Comment by PakhiAgarwal** [comment]
Now, I am having this:

replicates = get_replicates(study[0], study[1])
TypeError: get_replicates() missing 1 required positional argument: 'label'

What does 'label' signify in get_replicates() and what argument am I supposed to provide here?

**Comment by PakhiAgarwal** [comment]
@rjzupkoii I had made changes. Please have a look. The script is working now.

**Pull request merged**